content/id/docs/tasks/tools/kubeadm/high-availability.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Membuat Klaster dengan Ketersediaan Tinggi menggunakan kubeadm
```

Comment:
nit: need consistency here: membuat or membangun (from the _index.md)

Reply:
Ah, I updated the index but forgot about the title. Fixed in bc5c6c5.